airflow/provider.yaml.schema.json

```diff
@@ -231,8 +234,26 @@
       }
     },
     "additional-extras": {
-      "type": "object",
-      "description": "Additional extras that the provider should have"
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "description": "Name of the extra",
+            "type": "string"
+          },
+          "extras": {
```

Comment:
This field name is misleading. What do you think about `requirements` or `dependencies`?

Reply:
Yeah. Agree. It is - I realised about it yesterday - will fix it tomorrow